### ISSUE TITLE
CreateObject IfcCartesianTransformationOperator3DnonUniform doesn't seem to work properly

### ISSUE THREAD

**Issue by PatriceMoreau56** [opensourceBIM/BIMserver#949]
Hi,

I'm trying to create an IfcCartesianTransformationOperator3DnonUniform object with the low level level interface :
```
oid = createObject(transactionId, "IfcCartesianTransformationOperator3DnonUniform", false);
setDoubleAttribute(transactionId, oid, "Scale", 1D);
setDoubleAttribute(transactionId, oid, "Scale2", 2D);
setDoubleAttribute(transactionId, oid, "Scale3", 1D);
``` 
I reference it inside a MappedItem object.

The commit is successful and the BIMvie.ws reports the correct number of occurances of IfcCartesianTransformationOperator3DnonUniform. 
However, when I try to display all the occurances , none of them are displayed, not even the OID.
The object is also not display inside the MappedItem.

![2019-03-13 16_51_09-Clipboard](https://user-images.githubusercontent.com/34338280/54297155-948ace80-45b6-11e9-9b91-49bbae4528f3.png)
![2019-03-13 16_50_17-BIMvie ws - Browser](https://user-images.githubusercontent.com/34338280/54297437-24307d00-45b7-11e9-85e0-a762cb9b3acd.png)

I have also tried to export the revision as an IFC file. The Ifc file doesn't seem to contain the IfcCartesianTransformationOperator3DnonUniform object.

Thank you for your reply.

Edit 
Actually, it seems that you can't create an object if its type doesn't already exists.

----
Configuration:
Bimserver 					1.5.125
BIMServer JavaScript API	                0.0.155
BIMsurfer				        	0.0.64
BIMvie.ws					0.0.141
BinarySerializers			        0.0.57
IfcOpenShellPlugin			        0.5.51
IfcPlugins					        0.0.57

**Comment by rubendel** [comment]
> Actually, it seems that you can't create an object if its type doesn't already exists.

Nice catch! I can reproduce this. Will link to this issue when fixed.

**Comment by rubendel** [comment]
Fixed in git master, will be in 1.5.126
